tests/test_market.py

```diff
@@ -40,7 +40,7 @@ def test_parse_market_state():
 
 def test_order_book_iterator():
     """Test order book parsing."""
-    with open("tests/ask_order_binary.txt", "r") as input_file:
+    with open("tests/binary/ask_order_binary.txt", "r") as input_file:
```

Comment:
nit: Can you use `.bin` extension for these files.

Reply:
Changed to `.bin`